docs/static/dead-letter-queues.asciidoc

```diff
@@ -0,0 +1,155 @@
+[[dead-letter-queues]]
+=== Dead Letter Queues
+
+//REVIEWERS: I had to install logstash-input-dead_letter_queue. Is it not bundled with the alpha2 release?
+
```

Comment:
I think it will be installed with alpha3, I don't think it was added to alpha2 - @suyograo ?